core/loadpoint.go

```diff
@@ -1118,7 +1118,7 @@ func (lp *LoadPoint) pvMaxCurrent(mode api.ChargeMode, sitePower float64, batter
 	}
 
 	// in MinPV mode return at least minCurrent
-	if (mode == api.ModeMinPV || batteryBuffered) && targetCurrent < minCurrent {
+	if (mode == api.ModeMinPV || batteryBuffered || lp.climateActive()) && targetCurrent < minCurrent {
```

Comment:
Wozu soll das dienen? Der Strom wird später ohnehin gesetzt wenn climateActive?

Reply:
Wo?
Wenn hier nicht zurückgegeben wird läuft er in den PV-Abschalttimer rein.

Reply:
Der Kommentar in 1120 ist und war natürlich nicht mehr vollumfänglich.